### ISSUE TITLE
Container startup error

### ISSUE THREAD

**Issue by pickeld** [lolouk44/xiaomi_mi_scale#31]
```
Traceback (most recent call last):
  File "/opt/miscale/Xiaomi_Scale.py", line 344, in <module>
    main()
  File "/opt/miscale/Xiaomi_Scale.py", line 312, in main
    if MQTT_DISCOVERY.lower() in ['true', '1', 'y', 'yes']:
AttributeError: 'bool' object has no attribute 'lower'
```

**Comment by lolouk44** [comment]
Can you confirm your setup? Ideally if you could fill the template it would help...

**Comment by lolouk44** [comment]
fixed in 0.1.10
